core/controllers/reader.py

```diff
@@ -332,9 +332,6 @@ def post(self, unused_exploration_id):
         old_state = exp_domain.State.from_dict(self.payload.get('old_state'))
         # The learner's raw answer.
         answer = self.payload.get('answer')
-        # The learner's parameter values.
```

Comment:
Those parameter values weren't required. Leaving it like that caused Lint errors (unused variable). 
Other than this, the line params['answer'] = answer wasn't required either as the 'params' dict already had that key.